tools/generator/templates/player/caml/api.ml.jinja2

```diff
@@ -0,0 +1,39 @@
+{# Constants #}
+{% for constant in game.constant %}
+{{ constant.cst_comment|caml_comment(doc=True) }}
+let {{ constant.cst_name|lower }} = {{ constant.cst_val }}
+
```

Comment:
Generally we put the newline after the for, so that it avoids leaving empty lines afterward (especially useful at the end of the file)

Reply:
I know, but here it would add a empty line at the start of the for.